.eslintrc

```diff
@@ -20,14 +20,17 @@
     "max-len": [
       "warn",
       {
-        "code": 150,
+        "code": 350,
         "tabWidth": 2,
         "ignoreComments": true
       }
     ],
-    "react/prop-types": "error",
+    "react/prop-types": "off",
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "no-underscore-dangle": "off",
+    "no-console": "off",
+    "react/destructuring-assignment": [1, "never"],
+    "no-plusplus": [2, { "allowForLoopAfterthoughts": true }],
```

Comment:
I think it's better to use +=1 rather than ++, even in a for loop. It helps with readability as well, which is worth the extra character.

Reply:
For for loops, it's better to allow both just for dev's existing behaviour. I can see this being disabled if not by us, but by consumers of the project anyway.